### PR TITLE
Strip href from links and output images as [image]

### PR DESCRIPTION
**Pull request by brycelambert**

### CODE REVIEW

**Comment by mkostick** [comment]
Can you remove the `deploy` key from `.travis.yml`?  I want to get CI set up on this, but we don't want to publish to RubyGems.

**Review by drake-yesware — APPROVED** [review]

File: spec/examples/full_email.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-http://localhost/home	16 December 2015
+[image]	16 December 2015
```
Comment (mkostick) [inline spec/examples/full_email.txt:1]: Why is this converted to `[image]`?  That doesn't seem right.

File: spec/examples/full_email.html [hunk 2]
```diff
@@ -79,7 +79,7 @@ <h2 style="font-family: Times New Roman; font-size: 1.8; font-weight: normal; li
   <h3 style="font-family: Times New Roman; font-size: 18px; font-weight: normal; line-height: 2em; margin: 10px 0 0">Your achievements</h3>
   <table class="current-usage with-icon-left" style="border-collapse: collapse; border-spacing: 0; margin-bottom: 20px; margin-top: 20px; width: 100%">
     <tr>
-      <th style="border: none; font-family: Times New Roman; font-size: 14px; font-weight: bold; margin: 0; padding: 0; text-align: left; vertical-align: middle; width: 50px" align="left" valign="middle"><img src="test.png"></th>
+      <th style="border: none; font-family: Times New Roman; font-size: 14px; font-weight: bold; margin: 0; padding: 0; text-align: left; vertical-align: middle; width: 50px" align="left" valign="middle"></th>
```
Comment (mkostick) [inline spec/examples/full_email.html:82]: Shouldn't this stay as-is, with the expected value updated to include `[image]`?

File: spec/examples/full_email.txt (hunk 1 above)
Comment (mkostick) [reply spec/examples/full_email.txt:1]: Nevermind.  I didn't see where this originated in `full_email.html`.

**Comment by mkostick** [comment]
Done with my review

File: spec/examples/full_email.txt (hunk 1 above)
Comment (brycelambert) [reply spec/examples/full_email.txt:1]: Thanks for pointing that out. In the html, this is a link-wrapped image.

https://github.com/Yesware/html2text_ruby/blob/master/spec/examples/full_email.html#L29

Lexi asked that urls be stripped from link-wrapped images.

File: spec/examples/full_email.html (hunk 2 above)
Comment (brycelambert) [reply spec/examples/full_email.html:82]: Sure I can do that. My thought was that we had enough tests at work here so I took some liberties.

Comment (mkostick) [reply spec/examples/full_email.html:82]: Ah, yes, after reviewing the rest of the pull, I agree.

**Review by mkostick — APPROVED** [review]
G2G

**Pull request merged**